### ISSUE TITLE
Tool Box reverts back to individual item when 'changed'

### ISSUE THREAD

**Issue by JigglesTheFett** [AnimeniacYuuto/InventoryTools#4]
Howdy! First off, I absolutely love this mod, the tool boxes especially. They're fantastic.

Now, I've come across a few things that I haven't seen an issue for. If there is already something and I missed it, please forgive me.

Some items, when in the tool box, when placed into another inventory or 'changed' in some manner, revert the entire thing back to just that item.

Off the top of my head, I know of two such items.

The sonic screwdriver from the Tardis Mod. If it is in the box and your active tool, and you place it in one of the ports and change something on it, when you pull it back out it will just be the sonic screwdriver, no longer the tool boxed version.

The remote orderer from Logistics pipes. I had to resync my orderer to a pipe, and once I did that it reverted back to its un-toolboxed state thus losing all the rest of the stuff inside.

I've been afraid to try anything else, but I have a feeling that if it's not just those two items causing problems, then things like Ender Pouches would revert if you changed which Ender Chest they are linked to, etc.

**Comment by AnimeniacYuuto** [comment]
Yeah this is a known problem (even mentioned on the curse forge page). The current way the tool box works the tool box data is added to the item's NBT data, which many mods reset when they change an item cause they assume they are the only ones altering it. I have an idea for how I might do this better to avoid this issue but it probably won't happen for 1.7.10
